Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM registry.centos.org/centos/centos:7
+MAINTAINER Arunkumar Srisailapathi <asrisail@redhat.com>
+
+RUN useradd 3scale
+# Install deps for cryptography:
+#  https://cryptography.io/en/latest/installation/#rhel-centos
+RUN yum install -y gcc &&\
+    yum install -y epel-release &&\
+    yum install -y python34-pip &&\
+    yum install -y redhat-rpm-config gcc libffi-devel python34-devel openssl-devel &&\
+    yum clean all
+
+RUN mkdir -p /3scale
+COPY ./ /3scale
```

Comment:
Can be `/3scale` deleted after installation? I see workdir to be changed bellow, but are there any files that are needed after installation?

Reply:
Work directory is 3scale right?